webui/react/src/components/ClusterMessage.test.tsx

```diff
@@ -0,0 +1,39 @@
+// components/Counter.test.tsx
+import { render, screen } from '@testing-library/react';
+
+import { ClusterMessage } from 'stores/determinedInfo';
+
+import ClusterMessageBanner from './ClusterMessage';
+
+const setUp = (msg?: ClusterMessage) => {
+  render(<ClusterMessageBanner message={msg} />); // render arbitrary components
+};
+
+const msg = 'this is a test msg for use in the ClusterMessageBanner test';
+
+describe('ClusterMessageBanner', () => {
+  it('should have a banner with cluster message text when there is a cluster message', () => {
+    const time = new Date();
+    const testMsg = { createdTime: time, endTime: time, message: msg, startTime: time };
+
+    setUp(testMsg);
+    // make sure these components exist.
+    expect(screen.getByTestId('admin-msg')).toBeInTheDocument();
+    expect(screen.getByTestId('cluster-msg')).toBeInTheDocument();
+
+    // make sure the cluster message is visible, but the admin msg is not.
+    expect(screen.getByText(msg)).toBeInTheDocument();
```

Comment:
We care about the content as opposed to the element. In this test case we care about the text content as well, os we should add a check for the actual content.
@amandavialva01 @keita-determined

Reply:
use testID then get by text inside the test ID element